src/Testing/MakesGraphQLRequests.php

```diff
@@ -101,6 +101,39 @@ protected function multipartGraphQL(array $parameters, array $files, array $head
         );
     }
 
+    /**
+     * Send a multipart form request to GraphQL in array format. For mutations that are
+     * very large or contain many files, it becomes much easier to work with array instead of json.
```

Comment:
```suggestion
     * Send a multipart form request to GraphQL.
```

I agree with what is written, but prefer to let the usefulness of the method speak for itself ;)

An actual usage example would be more useful.